data/json/items/comestibles/offal_dishes.json

```
@@ -187,23 +187,23 @@
     "id": "grilled_sweetbread",
     "type": "COMESTIBLE",
     "copy-from": "sweetbread_cooked",
-    "name": "grilled sweetbread",
+    "name": { "str": "grilled sweetbread" },
     "fun": 6,
     "description": "Not sweet, like the name would suggest, but delicious all the same!"
   },
   {
     "id": "canned_liver",
     "type": "COMESTIBLE",
     "copy-from": "liver_cooked",
-    "name": "canned liver",
-    "description": "Livers preserved in a can.  Chock-full of B vitamins!",
+    "name": { "str": "canned liver" },
+    "description": "Livers preserved in a can.  chock-full of B vitamins!",
```

```suggestion
    "description": "Livers preserved in a can.  Chock full of B vitamins!",
```
Make it the same as the other Chock-full edit?